docs/getting_started/sso/okta.md

```diff
@@ -0,0 +1,53 @@
+# Configure Single Sign-on with Okta
+
+In order to configure Okta with SEKOIA.IO, the following steps must be done:
+
+1. Create a new application on Okta.
+2. Connect to SEKOIA.IO and add a new domain that belongs to your community and wait for its validation.
+3. Configure OpenID Connect in SEKOIA.IO (see associated documentation [Single Sign-On With OpenID Connect](../SSO_openid_connect.md))
+
+
+## Create
+
+1. Follow the menu to "admin console" > "Applications" > "Applications"
+2. Use the "Create App Integration" function to start the integration:
+    - As "Sign-in method" choose "OIDC - OpenID Connect".
+    - As "Application type" choose "Web Application".
+3. Once redirected to the "New Web App Integration", be sure to select:
+    - Grant type: "Client acting on behalf of a user" > "Authorization Code"
+    - Sign-in redirect URIs: `https://app.sekoia.io/user/callback`
+4. In "Assignments" > "Controlled access", you can limit access to specific groups of users (during tests you can use the option "Allow everyone in your organization to access").
+
+## Configure
+
+Once your app has been created:
+
+1. Save the "Client ID" & "Client secret" for later use.
+2. Save your Okta domain (go to the user menu in the top right corner of the Okta interface, you will find it under your email address), you will need it to provide it in SEKOIA.IO "Authentication provider base URL" field: `https://${yourOktaDomain}/` (More info
```

Comment:
```suggestion
2. Save your Okta domain. You can find this by navigating to the user menu in the top right corner of the Okta interface, and it will be listed under your email address. You will need it to provide it in SEKOIA.IO "Authentication provider base URL" field: `https://${yourOktaDomain}/`
```